services/api/src/typeDefs.js

```diff
@@ -278,16 +283,22 @@ const typeDefs = gql`
     uptimeRobotStatusPageId: String
   }
 
-  type Openshift {
+  interface DeploymentTarget {
+    id: Int
+    name: String
+    created: String
+  }
+
+  type Kubernetes implements DeploymentTarget {
     id: Int
     name: String
-    consoleUrl: String
-    token: String
-    routerPattern: String
-    projectUser: String
-    sshHost: String
-    sshPort: String
     created: String
+    apiType: KubernetesApiType
+    apiEndpoint: string
+    apiToken: string
```

Comment:
I think with the new ssh-portal system https://github.com/amazeeio/lagoon/pull/2179 we will not need any apiEndpoint or apiToken stored anymore, as everything happens now inside the cluster itself.
Can you confirm @smlx?

Reply:
Yes, however `ssh-portal` cannot run in the remote cluster until https://github.com/amazeeio/lagoon/issues/2181 is implemented.